src/ontology/uberon-edit.obo

```diff
@@ -224736,18 +224750,40 @@ relationship: dc-contributor http://orcid.org/0000-0001-6677-8489
 property_value: dcterms-date 2023-02-10T14:54:06Z xsd:dateTime
 
 [Term]
-id: UBERON:8480047
-name: retroauricular region
-def: "The retroauricular region is the area located behind the auricle of the outer ear." [http://orcid.org/0000-0002-0819-0473]
-synonym: "retro-auricular area" EXACT [PMID:22592943]
-synonym: "retro-auricular part of head" EXACT [FMA:322167]
-synonym: "retro-auricular region" EXACT [PMID:33988139]
-synonym: "retroauricular area" EXACT [PMID:29470606]
-is_a: UBERON:0000061 ! anatomical structure
-relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
-relationship: part_of UBERON:0000033 ! head
-property_value: curator_notes "Exists at least in all mammals. https://www.imaios.com/en/vet-Anatomy/Vet-Anatomical-Part/Retroauricular-region" xsd:string
-property_value: dcterms-date 2023-02-09T09:39:18Z xsd:dateTime
+id: UBERON:8600004
+name: visceral muscle tissue
+def: "A muscle tissue that is associated with internal organs (viscera), except the heart." [BTO:0002780, CLAO:0001295]
+comment: It is quite common to refer to visceral muscles as smooth muscles, by analogy with vertebrates; however most visceral muscles in arthropods are actually striated. {xref="PMID:12132390", xref="PMID:8720463"}
+is_a: UBERON:0002385 ! muscle tissue
+relationship: dc-contributor http://orcid.org/0000-0001-6677-8489
+property_value: dcterms-date 2023-02-16T10:17:46Z xsd:dateTime
+
+[Term]
+id: UBERON:8600005
+name: visceral smooth muscle tissue
+def: "A smooth muscle tissue that is associated with internal organs (viscera)." [PMID:12132390, PMID:33729000]
+is_a: UBERON:0001135 ! smooth muscle tissue
+is_a: UBERON:8600004 ! visceral muscle tissue
+relationship: dc-contributor http://orcid.org/0000-0001-6677-8489
+property_value: dcterms-date 2023-02-16T10:32:12Z xsd:dateTime
+
+[Term]
+id: UBERON:8600006
+name: visceral striated muscle tissue
+def: "A striated muscle tissue that is associated with internal organs (viscera), except the heart." [https://pressbooks.pub/rbtallitsch/chapter/chapter-6-muscle-tissue/, PMID:8720463]
+synonym: "striated visceral muscle tissue" EXACT []
+is_a: UBERON:0002036 ! striated muscle tissue
+is_a: UBERON:8600004 ! visceral muscle tissue
+relationship: dc-contributor http://orcid.org/0000-0001-6677-8489
+property_value: dcterms-date 2023-02-16T10:34:43Z xsd:dateTime
+
+[Term]
+id: UBERON:8600007
+name: transversely striated visceral muscle tissue
```

Comment:
For label consistency, consider labelling as 'visceral transversely striated muscle tissue' and making this a syn if necessary.